### ISSUE TITLE
Refuses to work for longer periods of time?

### ISSUE THREAD

**Issue by darkblood159** [greg-ynx/NexusDownloadFlow-2022#2]
Hey I was trying this out and I seen to have gotten it to work. But leaving it on for around 10m or so it will often start to fail/mess up or I have to start it up by doing a part manually. Sometimes it just straight up refuses to work unless I restart the program. Any way I can fix this up?

**Comment by greg-ynx** [comment]
Hi! I think that your issue is caused by the screenshot with the cursor on the button, or something is poping into your screen...

Does the script close itself or show anything (like freeze) when this issue happen?

**Comment by greg-ynx** [comment]
Hi, I hope you are doing well :)
I think that I've found the origin of your issue, do you have more than one monitor/display?
NexusDownloadFlow2022 only works on your "main" monitor.

However, next version of NDF will work with multiscreen setups ;)

Tell me if you have multiple display in order to know if this is really the origin of your issue.
Regards

**Comment by darkblood159** [comment]
I apologize for not responding earlier. I do have two monitors.

**Comment by greg-ynx** [comment]
No problem 😄 
If this issue is caused by the fact that you use NDF on the "wrong" monitor, don't worry it is fixed for the `2.0.0` version 😉

**Comment by greg-ynx** [comment]
I don't know if you will have to download new mod list but the new version of NDF was just released tell me if your issue ever persist on this version 😄

**Comment by Arhkeis** [comment]
Hello, I decided to write here instead of creating a new request, since the problem is similar to mine.
The essence of the problem, as the author indicated above, works for about 10 minutes.
But I get the error Exception Processing Message 0xc0000005 - Unexpected Parameters.
I don’t know what this is connected with, I have 1 monitor.
If necessary, I can take a screenshot and send it to you.

**Comment by greg-ynx** [comment]
Hi @Arhkeis! Are you using the 2.0.0 version? If yes, please share your log file. And yes I would like to see a screenshot, anything that will help me fixing this issue 😅

**Comment by Arhkeis** [comment]
Yes, I just downloaded your program yesterday and tried it. The program itself is very convenient for me, I like it. But the error spoils the impression a little)
But it’s very likely that the problem is not on your part, because I haven’t used Wabbajack for quite a long time and now there’s a Pre-Release version.
![Screenshot 2023-11-07 020023](https://github.com/greg-ynx/NexusDownloadFlow/assets/96066207/78fd11d0-e733-4a44-8cce-6037f1ab6445)
[2023_11_07_ndf.log](https://github.com/greg-ynx/NexusDownloadFlow/files/13279812/2023_11_07_ndf.log)

**Comment by greg-ynx** [comment]
Thank you for the screenshot! As you can see, the error is raised from `wabbajack.exe` which is the Wabbajack executable. Your issue does not come from NDF. As you've said it might come from Wabbajack's recent release. I hope they will fix it. Good luck!

**Comment by Arhkeis** [comment]
> Thank you for the screenshot! As you can see, the error is raised from `wabbajack.exe` which is the Wabbajack executable. Your issue does not come from NDF. As you've said it might come from Wabbajack's recent release. I hope they will fix it. Good luck!

Thank you for your prompt response!

**Comment by greg-ynx** [comment]
I will close this issue for inactivity. There wont be any patch on 2022v1.0.0 version of NDF.

Please use current version.
